RELEASE-PROCESS.md

```diff
@@ -0,0 +1,53 @@
+# Release Process
+
+External facing [Traffic Director](https://cloud.google.com/traffic-director)
+documentation for specific features tend to use the wallet example to showcase
+the feature better. Here is one such [example][wallet-example-in-td-docs] where
+a specific branch of the wallet example is used.
+
+[wallet-example-in-td-docs]:
+https://cloud.google.com/traffic-director/docs/proxyless-configure-advanced-traffic-management#verify-routing-configuration
+
+# When to make a new release
+Whenever we have a new iteration of the public documentation, referencing the
+wallet example to showcase new set of functionality, we should consider making a
```

Comment:
"new set of functionality" is kind of subjective. Is adding a new command line flag new functionality? If the new functionality is only on the control plane side so there is no code change in the example code but update of docs do we still need to make a new release? Are there cases where we make changes on the latest release branch instead of making a new release?

Reply:
"should consider" -> _must_

We don't want to change the 1.0.x branch except for bug fixes.